hack/csi_test/cmd/main.go

```diff
@@ -9,8 +9,8 @@ import (
 
 var (
 	endpoint    = flag.String("endpoint", "/tmp/csi.sock", "CSI endpoint")
-	host        = flag.String("host", "localhost", "Host machine name")
-	volume      = flag.String("volume", "", "Volume name")
+	shareAddr   = flag.String("shareAddr", "localhost", "Address of the share")
+	shareName   = flag.String("volume", "", "Volume name")
```

Comment:
also update the flag name and description too?  Volume is ambiguous in the CSI context because it could mean the CSI volume name.